modular_sojourn/area_blurbs.dm

```diff
@@ -0,0 +1,157 @@
+	narrate = "Abanadond outpost filled with trash and spiders."
+
+/area/nadezhda/dungeon/outside/zoo
+	narrate = "Cold damn air lingers as the now dorment building stands with unknown inside."
+
+/area/nadezhda/dungeon/outside/prepper
+	narrate = "A valt build into the moutain side housing robotic and treasers inside. That is unless the roaches or spiders got here first..."
+
+/area/nadezhda/dungeon/outside/prepper/vault/entryway
+	narrate = "A complex of housing, storage and power all left to ruin."
+
+/area/nadezhda/dungeon/outside/smuggler_zone
+	narrate = "A small outpost for pirates and smugglers alike."
+
+/area/nadezhda/dungeon/outside/smuggler_zone_u
+	narrate = "The inside of the smugglers outpost, lively with the smell of cheap drugs and cheaper booze."
+
+/area/nadezhda/outside/one_star
+	narrate = "A imposing office center in the middle of noware, sorting Greyson robotics portecting its treasers inside."
+
+/area/nadezhda/outside/dcave
+	narrate = "A common cave filled with wildlife."
+
+/area/nadezhda/outside/pond
+	narrate = "A large pond housing life from frogs to croaker lords alike."
+
+/area/nadezhda/outside/scave
+	narrate = "A small cave that is common to see spiders lerking inside."
+
+/area/nadezhda/outside/forest/swamp_hut
+	narrate = "A small hut with a farm in the hostile swamp terrain."
+
+/area/nadezhda/outside/forest/hunting_lodge_shed_dark
+	narrate = "The smell of farm animals and dust fills the area."
+
+/area/nadezhda/outside/mountainsolars
+	narrate = "Being so high up makes the air thin, and the sun beam down harder."
+
+// SUBSTATIONS (Subtype of maint, that should let them serve as shielded area during radstorm)
+
+/area/nadezhda/maintenance/substation
+	narrate = "One of the many substations around the colony seeing the SMES storage and power net sceners."
+
+//Command
+
+/area/nadezhda/command/meeting_room
+	narrate = "A place for behind close door meetings to get things done, or argue for hours in..."
+
+/area/nadezhda/command/armory
+	narrate = "Clean fire arms and armor for when marshals and blackshield fails..."
+
+/area/nadezhda/command/panic_room
+	narrate = "A vault for the command personal unable to aid in cases of colony unrest or attack."
+
+/area/nadezhda/command/tcommsat
+	narrate = "Cold air and plinking sounds of the telecoms log, broadcast and send messages at high speed."
+
+/area/nadezhda/command/courtroom
+	narrate = "Laws only applies if you have the authority to enforce it. You find the authority here."
+
+//Crew Quarters
+
+/area/nadezhda/crew_quarters/kitchen
+	narrate = "All the cooking tools, and equiment needed for a fine meal."
+
+/area/nadezhda/crew_quarters/bar
+	narrate = "The long bar and music box singles the relaxing atmosphere."
+
+/area/nadezhda/crew_quarters/janitor
+	narrate = "The smell of space cleaner stings the nose."
+
+/area/nadezhda/security/armory
+	narrate = "Weapons line the shelfs and the glimer reflects off the armor."
```

Comment:
```suggestion
	narrate = "Weapons line the shelves in perfect order and even more perfect condition. Ammunition is neatly kept separate. The autolathe clicks and pistons in anticipation for the guns and ammunition it will print on yet another busy day..."
```